src/components/data_search/DatasetSearchTable.js

```diff
@@ -351,81 +370,91 @@ export const DatasetSearchTable = (props) => {
-                  <h1>There are no datasets that fit these criteria.</h1>
-                </Box>
-              );
-            } else {
-              return (
-                <CollapsibleTable
-                  data={tableData}
-                  selected={selected}
-                  selectHandler={selectHandler}
-                  expandHandler={expandHandler}
-                  collapseHandler={collapseHandler}
-                  summary='faceted study search table'
-                />
-              );
-            }
-          })()}
+        <Box sx={{display: 'flex', flexDirection: 'row', paddingTop: '2em'}}>
+          <Box sx={{width: '14%', padding: '0 1em'}}>
+            <DatasetFilterList datasets={datasets} filters={filters} filterHandler={filterHandler} searchRef={searchRef}/>
+          </Box>
+          <Box sx={{width: '85%', padding: '0 1em'}}>
+            {(() => {
+              if (isEmpty(datasets)) {
+                return (
+                  <Box sx={{
+                    display: 'flex',
+                    flexDirection: 'column', alignItems: 'center', justifyContent: 'center', height: '100%' }}>
+                    <h1>No datasets registered for this library.</h1>
+                  </Box>
+                );
+              } else if (isEmpty(filtered)) {
+                return (
+                  <Box sx={{
+                    display: 'flex',
+                    flexDirection: 'column', alignItems: 'center', justifyContent: 'center', height: '100%' }}>
+                    <h1>There are no datasets that fit these criteria.</h1>
+                  </Box>
+                );
+              } else {
+                return (
+                  <CollapsibleTable
+                    data={tableData}
+                    selected={selected}
+                    selectHandler={selectHandler}
+                    expandHandler={expandHandler}
+                    collapseHandler={collapseHandler}
+                    summary='faceted study search table'
+                  />
+                );
+              }
+            })()}
+          </Box>
         </Box>
-      </Box>
-      <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', padding: '2em 4em' }}>
-        {
-          !isEmpty(datasets) &&
+        <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', padding: '2em 4em' }}>
+          {
+            !isEmpty(datasets) &&
           <Button variant="contained" onClick={applyForAccess} sx={{ transform: 'scale(1.5)' }} >
             Apply for Access
           </Button>
-        }
+          }
+        </Box>
       </Box>
-    </Box>
+      {
+        showTranslatedDULModal &&
+        <TranslatedDulModal
+          showModal={showTranslatedDULModal}
+          dataUse={dataUse}
+          onCloseRequest={()=>setShowTranslatedDULModal(false)}
+        />
+      }
```

Comment:
Since we are using material UI on the newer pages, I think we should switch this modal to follow material UI or create a new material UI modal for the translated DUL.

Reply:
Sure @fboulnois , I can update the code to use material ui.

Reply:
@fboulnois  I have added an `UpdatedBaseModal `using the modal from `@material ui`. I have the appearance of the modal intact.

Reply:
I think we want to keep the styling consistent between the data library and the modals. For example, see `modals/ConfirmationDialog.js` for how this could be implement using only Material UI. I have a PR which uses that modal here: https://github.com/DataBiosphere/duos-ui/pull/2439/files

One problem with the older modals is they do not respect z-ordering, and are thus often inaccessible or unreachable if another modals or the header is at the top of the screen.

Reply:
@fboulnois  I made some updates. Let me know what you think.

Reply:
Looks great, thank you! 👍